server/utils/checkout.js

```diff
@@ -5,7 +5,7 @@ let checkoutId;
 async function checkoutStripe(cart) {
   const stripeCart = cart.map(item => ({
     name: item.product.name,
-    amount: 100 || item.product.cost,
+    amount: item.product.cost || 99999,
     currency: 'usd',
```

Comment:
I think I like this. So if a malicious person tried to set the cost to 0 then it would be 99999. They owe us that much anyway right?

Reply:
When I wrote it I was thinking along the lines of an admin forgetting to set  price when adding new items, but I supposed that situation would be better handled by seqeulize validations. Works well for that situation too.